plugins/modules/elastic_integration_info.py

```diff
@@ -45,19 +45,17 @@ def main():
 
     results['changed'] = False
     
-    integration_name = module.params.get('integration_name')
-    
+    integration_title = module.params.get('integration_title')
+  
     kibana = Kibana(module)
       
-    integration_object = kibana.check_integration(integration_name)
+    integration_object = kibana.check_integration(integration_title)
```

Comment:
`check_integration` was changed so the value of `integration_object` here wouldn't be the same now that we aren't getting the value of `response` before leaving `check_integration`. Is this intended?

Reply:
this was intended